Library/build.gradle

```diff
@@ -66,9 +66,9 @@ android {
             exclude group: 'org.jetbrains.kotlin', module: 'kotlin-stdlib-common'
         }
 
-        testImplementation "org.mockito:mockito-core:${versions.mockito2}"
-        testImplementation "org.mockito.kotlin:mockito-kotlin:${versions.mockitokotlin}"
+        testImplementation "io.mockk:mockk:${versions.mockk}"
         testImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:${versions.kotlinx}"
+        testImplementation "org.slf4j:slf4j-jdk14:${versions.slf4j}"
```

Comment:
I was a bit 🤨 at the JDK14 reference, but checking their artifacts, it's apparently for JDK _1.4_ not _14_. TIL.

Reply:
Yeah, I was thrown off a bit by this.